packages/react/src/SelectPanel/SelectPanel.module.css

```diff
@@ -134,17 +134,61 @@
 
 .ResponsiveFooter {
   display: none;
+  align-items: center;
   padding: var(--base-size-16);
   gap: var(--stack-gap-condensed);
-  justify-content: right;
+  justify-content: center;
 
-  @media screen and (--viewportRange-narrow) {
+  &:where([data-display-footer='always']) {
     display: flex;
-    gap: var(--stack-gap-condensed);
-    justify-content: right;
   }
 
-  &:where([data-variant='modal'] &) {
```

Comment:
The idea with this change is that the footer layout logic is colocated on the `tsx`, so the css already receives which layout it has to render, without it having to check the variants or anything like that.